src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -740,7 +790,10 @@
            hostprivkey)))
 
     :else
-    (generate-master-ssl-files! settings certname ca-settings)))
+    (generate-master-ssl-files! settings certname ca-settings))
+  ;; Generate a file containing serial numbers of the configured compiler master certificates, if any.
+  (generate-compile-master-serials ca-settings)
```

Comment:
This line should aligned with the one above it

Reply:
I wanted it to be outside of the above conditional and execute every time since we don't know if the infra-nodes changed since the last time.